ksql-engine/src/main/java/io/confluent/ksql/KsqlContext.java

```diff
@@ -38,9 +41,19 @@
   private final KsqlEngine ksqlEngine;
   private static final String APPLICATION_ID_OPTION_DEFAULT = "ksql_standalone_cli";
   private static final String KAFKA_BOOTSTRAP_SERVER_OPTION_DEFAULT = "localhost:9092";
-  private final KafkaTopicClientImpl topicClient;
+  private final AdminClient adminClient;
+  private final KafkaTopicClient topicClient;
+
+
+  public static KsqlContext create() {
+    return new KsqlContext();
```

Comment:
delegation, i.e, `create(Collections.emptyMap())`

Reply:
Good point, made the changes.